### ISSUE TITLE
Using any one of them Glide or Picasso

### ISSUE THREAD

**Issue by saketkumar95** [zulip/zulip-android#381]
**Actual Behaviour**

Both Glide and picasso is being used for fetching of images or anything.

**Expected Behaviour**

It would be better that any one can be used + apk size will be reduced too. 

**Would you like to work on the issue?**

I'm working on this. :)

**Comment by saketkumar95** [comment]
I guess it would be better to use Glide as it supports everything that of picasso + picasso doesnot support gifs. 
@kunall17 @Sam1301 Any suggestions?

**Comment by Sam1301** [comment]
@saketkumar95 yeah glide is better than picasso perf wise, you would prefer Glide for use. @kunall17 thoughts on this?

**Comment by kunall17** [comment]
We need to support GIF so we'll have to use glide!

**Comment by saketkumar95** [comment]
@kunall17 Yeah. I used Glide only. :)

**Comment by abhaymaniyar** [comment]
Is this issue resolved?

**Comment by Sam1301** [comment]
yeah, a pr has been made to address to issue.

**Comment by abdulwd** [comment]
Is this issue still open?

**Comment by meghalagrawal** [comment]
Is issue still open ?

**Comment by Sam1301** [comment]
@meghalagrawal I guess not, a pr has been opened for this.
